### PR TITLE
Pubstwo 1447

### PR DESCRIPTION
**Pull request by ayan-usgs**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,17 +39,20 @@ under PubsWarehouse_UI folder.  The contents of the file should look like so:
  # set the default path for the login page- for local development, it is '/login/'
  LOGIN_PAGE_PATH = '/login/'
  # verify ssl certificate for outside service calls
- VERIFY_CERT = True or False #Set to False for local development
+ VERIFY_CERT = True or False #Set to False for local development; can be a path to a certificate bundle
  #cache settings- see the documentation for flask-cache. For development, a cache type of simple works well with the development server.  redis is used on production
  #CACHE_CONFIG = {'CACHE_TYPE': 'simple'}
```
Comment (mbucknell) [inline README.md]: Should the comment line be here? It seems strange to put it in the README commented out.

File: README.md [hunk 2]
```diff
@@ -43,17 +39,20 @@ under PubsWarehouse_UI folder.  The contents of the file should look like so:
...
+ VERIFY_CERT = True or False #Set to False for local development; can be a path to a certificate bundle
  #cache settings- see the documentation for flask-cache. For development, a cache type of simple works well with the development server.  redis is used on production
  #CACHE_CONFIG = {'CACHE_TYPE': 'simple'}
  #set to the sciecebase folder id for the core publications warehouse SB folder
  SCIENCEBASE_PARENT_UUID = ''
  
- #The following should be removed when we completely retire the old MyPubs Angular app.
- OLD_MYPUBS_ENDPOINT = 'endpoint of the old mypubs application'
  
  #Assuming you use mvn test to install the lessc set LESS_BIN as follows:
```
Comment (mbucknell) [inline README.md]: We are no longer using mvn so this comment should be changed.

File: README.md [hunk 3]
```diff
@@ -43,17 +39,20 @@ under PubsWarehouse_UI folder.  The contents of the file should look like so:
...
+ ASSETS_DEBUG = False  # to disable compression of js and css set to True
+ ASSETS_AUTO_BUILD = False  # Local developers will typically set this to True in their instance/config.py.
  LESS_BIN = 'PROJECT_DIR/node_modules/less/bin/lessc'
```
Comment (mbucknell) [inline README.md]: LESS_BIN does not need to be in the instance/config.py. It is now set in the common config.py

File: README.md [hunk 4]
```diff
@@ -43,17 +39,20 @@ under PubsWarehouse_UI folder.  The contents of the file should look like so:
...
  #cache settings- see the documentation for flask-cache. For development, a cache type of simple works well with the development server.  redis is used on production
  #CACHE_CONFIG = {'CACHE_TYPE': 'simple'}
  #set to the sciecebase folder id for the core publications warehouse SB folder
  SCIENCEBASE_PARENT_UUID = ''
  
- #The following should be removed when we completely retire the old MyPubs Angular app.
- OLD_MYPUBS_ENDPOINT = 'endpoint of the old mypubs application'
  
  #Assuming you use mvn test to install the lessc set LESS_BIN as follows:
+ ASSETS_DEBUG = False  # to disable compression of js and css set to True
```
Comment (mbucknell) [inline README.md]: This is there twice. Should be removed from the top of the description of the instance/config.py

**Comment by ayan-usgs** [comment]
Completed requested changes.

**Pull request merged**